docs/installation/windows.md

```diff
@@ -9,122 +9,296 @@ parent = "smn_engine"
+
+4. Press "Next" to install the toolbox.
+
+    The installer presents you with options to customize the standard
+    installation. By default, the standard Docker Toolbox installation:
+  
+    * installs executables for the Docker tools in `C:\Program Files\Docker Toolbox` 
+    * updates any existing Virtual Box installation 
+    * adds a Docker Inc. folder to your program shortcuts
+    * updates your PATH environment variable
+    * adds desktop icons for the Docker CLI and Kitematic
 
-<iframe width="640" height="480" src="//www.youtube.com/embed/TjMU3bDX4vo?rel=0" frameborder="0" allowfullscreen></iframe>
+    This installation assumes the defaults are acceptable.
 
-## Installation
+5. Press "Next" until you reach the "Ready to Install" page.
 
-1. Download the latest release of the
-   [Docker for Windows Installer](https://github.com/boot2docker/windows-installer/releases/latest).
-2. Run the installer, which will install Docker Client for Windows, VirtualBox,
-   Git for Windows (MSYS-git), the boot2docker Linux ISO, and the Boot2Docker
-   management tool.
-   ![](/installation/images/windows-installer.png)
-3. Run the **Boot2Docker Start** shortcut from your Desktop or “Program Files →
-   Boot2Docker for Windows”.
-   The Start script will ask you to enter an ssh key passphrase - the simplest
-   (but least secure) is to just hit [Enter].
+     The system prompts you for your password.
+   
+     ![Install](/installation/images/win-page-6.png)
+     
+6. Press "Install"  to continue with the installation.
 
-4. The **Boot2Docker Start** will start a unix shell already configured to manage
-   Docker running inside the virtual machine. Run `docker version` to see
-   if it is working correctly:
+     When it completes, the installer provides you with some information you can
+     use to complete some common tasks.
+   
+     ![All finished](/installation/images/windows-finish.png)
+   
+7. Press "Close" to exit.
 
-![](/installation/images/windows-boot2docker-start.png)
 
-## Running Docker
+## Running a Docker Container
 
-> **Note:** if you are using a remote Docker daemon, such as Boot2Docker, 
-> then _do not_ type the `sudo` before the `docker` commands shown in the
-> documentation's examples.
+To run a Docker container, you:
 
-**Boot2Docker Start** will automatically start a shell with environment variables
-correctly set so you can start using Docker right away:
+* create a new (or start an existing) Docker virtual machine 
+* switch your environment to your new VM
+* use the `docker` client to create, load, and manage containers
 
-Let's try the `hello-world` example image. Run
+Once you create a machine, you can reuse it as often as you like. Like any
+Virtual Box VM, it maintains its configuration between uses.
 
-    $ docker run hello-world
+There are several ways to use the installed tools, from the Docker CLI applet or
+[from your shell](#from-your-shell).
+
+### From the Docker CLI
```

Comment:
Is it called "the Docker QuickStart Terminal" on windows as well? (You used that in the Mac docs now)